src/main/java/eu/yals/utils/git/GitRepoState.java

```diff
@@ -35,41 +37,56 @@
  * @since 2.0
  */
 @Slf4j
+@Data
 @Component
-class GitRepoState {
+public class GitRepoState {
     private static final String GIT_PROPERTIES_FILE = "git.properties";
 
     private final Properties gitProperties = new Properties();
 
     String commitIdAbbrev;          // =${git.commit.id.abbrev}
     String buildVersion;             // =${git.build.version}
+    String branch;                  //=${git.branch}
+    String buildHost;               //=${git.build.host}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)